File: app/src/main/java/ru/systempla/weatherapp/mvp/model/entity/SysRestModel.java
package ru.systempla.weatherapp.mvp.model.entity;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class SysRestModel {

    @SerializedName("type") public int type;
    @SerializedName("id") public int id;
    @SerializedName("message") public float message;
    @SerializedName("country") public String country;
    @SerializedName("sunrise") public long sunrise;
    @SerializedName("sunset") public long sunset;
}

File: app/src/main/java/ru/systempla/weatherapp/mvp/model/entity/SysRestModel.kt
package ru.systempla.weatherapp.mvp.model.entity

import com.google.gson.annotations.SerializedName

class SysRestModel {
    var type = 0

    var id = 0

    var message = 0f

    var country: String? = null

    var sunrise: Long = 0

    var sunset: Long = 0
}